### PR TITLE
Feature/persisting hmi capabilities temp branch

### PR DESCRIPTION
**Pull request by IGapchuk**
It's temporary PR for internal team review.

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Summary
this PR provides the changes according to the PR delivery PR review from the Livio team [3397](https://github.com/smartdevicelink/sdl_core/pull/3397)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: src/components/policy/policy_external/include/policy/policy_table/types.h [hunk 1]
```diff
@@ -485,7 +485,7 @@ struct ConsumerFriendlyMessages : CompositeType {
 
 struct ModuleMeta : CompositeType {
  public:
-  Optional<String<0, 250> > ccpu_version;
+  Optional<String<0, 500> > ccpu_version;
```
Comment (YarikMamykin) [inline src/components/policy/policy_external/include/policy/policy_table/types.h:488]: Why do we need this? Isn't that enough 250 chars for ccpu version?

Comment (LitvinenkoIra) [reply src/components/policy/policy_external/include/policy/policy_table/types.h:488]: @YarikMamykin according to the API

File: src/components/application_manager/src/hmi_capabilities_impl.cc [hunk 2]
```diff
@@ -1763,160 +1766,177 @@ void HMICapabilitiesImpl::PrepareUiJsonValueForSaving(
...
+
+    if (section_to_update == hmi_response::languages) {
+      save_hmi_capability_field_to_json(
+          hmi_response::languages, schema, ui_supported_languages(), out_node);
+    }
   }
 
-  if (helpers::in_range(sections_to_update, hmi_response::languages)) {
+  if (!system_capabilities->empty()) {
+    LOG4CXX_DEBUG(logger_, "KEK");
```
Comment (LitvinenkoIra) [inline src/components/application_manager/src/hmi_capabilities_impl.cc:1878]: @IGapchuk LOL

File: src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/get_system_info_response.cc [hunk 3]
```diff
@@ -82,12 +82,10 @@ const SystemInfo GetSystemInfoResponse::GetSystemInfo(
...
-  const uint32_t lang_code =
-      (*message_)[strings::msg_params]["language"].asUInt();
-  info.language = application_manager::MessageHelper::CommonLanguageToString(
-      static_cast<hmi_apis::Common_Language::eType>(lang_code));
-
-  hmi_capabilities_.OnSoftwareVersionReceived(info.ccpu_version);
+  const auto lang_code = static_cast<hmi_apis::Common_Language::eType>(
+      (*message_)[strings::msg_params]["language"].asUInt());
+  info.language =
+      application_manager::MessageHelper::CommonLanguageToString(lang_code);
```
Comment (LuxoftAKutsan) [inline src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/get_system_info_response.cc:88]: I can see `using namespace application_manager;` in this file. Remove `application_manager::`

File: src/components/application_manager/src/hmi_capabilities_impl.cc (hunk 2 above)
Comment (IGapchuk) [reply src/components/application_manager/src/hmi_capabilities_impl.cc]: @LitvinenkoIra Done in 7be5fccca8f77fc542cc64bcef5f6265e688b927

File: src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/get_system_info_response.cc (hunk 3 above)
Comment (IGapchuk) [reply src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/get_system_info_response.cc]: @LuxoftAKutsan Done in 6980ebfb521bacc653a2aecc3fdc432ffc05a8ea

**Pull request merged**